### ISSUE TITLE
Can't create more than one route to route table

### ISSUE THREAD

**Issue by kunalvudathu** [getmoto/moto#6305]
This is the snippet of the code I used:

```
mock_rtb_private_1_id = ec2.create_route_table(VpcId=mock_vpc_id)['RouteTable']['RouteTableId']
ec2.create_route(DestinationCidrBlock="10.0.0.8/8", NatGatewayId=mock_igw_id,
                     RouteTableId=mock_rtb_private_1_id)
ec2.create_route(DestinationCidrBlock="0.0.0.0/0", NatGatewayId=mock_nat_1['NatGatewayId'],
                     RouteTableId=mock_rtb_private_1_id)
```

I get the following error when I try to create the second route (line 3):

`botocore.exceptions.ClientError: An error occurred (RouteAlreadyExists) when calling the CreateRoute operation: The route identified by 0.0.0.0/0 already exists`

Thank you so much!

**Comment by bblommers** [comment]
Hi @kunalvudathu, which version of Moto are you using? Because this looks very similar to #6096  (fixed as of moto >= 4.1.6.dev17)

**Comment by kunalvudathu** [comment]
> Hi @kunalvudathu, which version of Moto are you using? Because this looks very similar to #6096 (fixed as of moto >= 4.1.6.dev17)

I was using 4.1.3, it works now! Thank you! Sorry about that.
